api/operator/v1alpha1/eventing_types.go

```diff
@@ -92,11 +92,15 @@ type Eventing struct {
 
 // EventingStatus defines the observed state of Eventing.
 type EventingStatus struct {
-	ActiveBackend     BackendType         `json:"activeBackend"`
-	BackendConfigHash int64               `json:"specHash"`
-	State             string              `json:"state"`
-	PublisherService  string              `json:"publisherService,omitempty"`
-	Conditions        []kmetav1.Condition `json:"conditions,omitempty"`
+	ActiveBackend     BackendType `json:"activeBackend"`
+	BackendConfigHash int64       `json:"specHash"`
+	// Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set
+	// when all the resources are deployed successfully and backend is connected.
```

Comment:
Is this correct right now? It might change together with: https://github.com/kyma-project/eventing-manager/issues/267

Reply:
this is independen from the issue. The owner of the issue should add more  details as the issue is completed, I'd say.